dev-lang/nasm/metadata.xml

```diff
@@ -15,5 +15,6 @@ Pentium, P6, MMX, 3DNow!, SSE and SSE2 opcodes, and has macro capability.
   </longdescription>
   <upstream>
     <remote-id type="sourceforge">nasm</remote-id>
+	<bugs-to>https://bugzilla.nasm.us/</bugs-to>
```

Comment:
I blame vim! (I didn't see that the original had spaces and my vim config uses tabs for those files, which is Gentoo's default I guess).